src/middlewared/middlewared/service/core_service.py

```diff
@@ -134,6 +135,32 @@ def get_tasks(self):
                 'frames': frames,
             }
 
+    @private
```

Comment:
This decorator is not needed for methods that are already private

Reply:
Whoops. Don't know why I did that.